### PR TITLE
New PR!

### PR DESCRIPTION
**Pull request by BensonCasey7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ wthr. utilizes the OpenWeatherMap API for real-time weather information, as well
 This test app can be viewed at https://floatwthr.firebaseapp.com/
 
 ## Dependencies
-- npm :)
+- npm :) <3
```
Comment (floatkeera) [inline README.md:8]: ok
?

Comment (BensonCasey7) [reply README.md:8]: Maybe

Comment (BensonCasey7) [reply README.md:8]: @floatkeera Yay

Comment (BensonCasey7) [reply README.md:8]: @BensonCasey7 woo

**Review by floatkeera — APPROVED** [review]

**Comment by BensonCasey7** [comment]
@floatkeera

**Pull request merged**